themes/default/content/blog/announcing-6-0-of-the-pulumi-aws-classic-provider/index.md

```diff
@@ -0,0 +1,66 @@
+
+You can see a full list of new resources and functions in [Release Notes](https://github.com/pulumi/pulumi-aws/releases/tag/v6.0.0).
+
+### SDK Size Optimization
+
+The AWS Classic provider contains more than 1,200 resources with over 12,000 properties between them. As a result, it’s expected that the SDK size of the provider has grown to be fairly large. With that in mind, we invested in some key areas to reduce the size of generated files. Depending on the runtime, the SDK is now up to 90% smaller than it used to be in v5 of the provider!
+
+![aws_sdk](aws_sdk.png)
+
+One particular optimization is related to `wafv2` resources, that were particularly difficult to use in Pulumi AWS v5 (issue [one](https://github.com/pulumi/pulumi-aws/issues/2276), [two](https://github.com/pulumi/pulumi-aws/issues/1117), [three](https://github.com/pulumi/pulumi-aws/issues/2250)). The new release includes a refactored `wafv2` module which uses recursive types. Using recursive types allowed us to remove a lot of excess generated code.
+
+Smaller SDK sizes mean faster downloads, snappier experience with editors and code completion tools, less time needed for `pulumi preview` and `pulumi up` operations, and smaller binaries, all of which contribute to a streamlined developer experience for Pulumi users.
+
+### Upstream Breaking Changes
+
+Pulumi AWS Classic 6.0 ships all improvements and bug fixes of upstream versions from 5.0.0 to 5.11.0. It also contains a number of upstream breaking changes. Please refer to the [changelog](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md) to navigate the entire list.
+
+### Fixes for Highly Upvoted Issues
+
+The new release ships fixes for many [highly upvoted issues along with  Pulumi-specific enhancements](https://github.com/pulumi/pulumi-aws/issues?q=label%3A6.0+sort%3Areactions-%2B1-desc+-label%3Akind%2Ftask+-label%3Akind%2Fepic+is%3Aclosed). This includes resolving [constant diff from wafv2 with no changes](https://github.com/pulumi/pulumi-aws/issues/1423) and [unexpected behavior of defaultTags](https://github.com/pulumi/pulumi-aws/issues/1655) that have been fixed and are available in the new release.
+
+### MaxItemsOne and Deprecations
+
+The 6.0 release removed a few resources that have been deprecated for a long while. All of those resources were replaced with newer versions containing a more logical organization of modules and resource names. The old resources have been deprecated for years, so it's unlikely that you were still using them. But in case you do, we applied automatic aliases to make migration as straightforward as possible.
```

Comment:
```suggestion
The 6.0 release removes several deprecated resources. All of those resources were replaced with newer versions containing a more logical organization of modules and resource names. The removed resources have been deprecated for years, so it's unlikely that you were still using them. But in case you do, we applied automatic aliases to make migration as straightforward as possible.
```